website/docs/guides/1.13.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1130-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.13.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+- You can now manage [MongoDB Atlas Search Node](https://www.mongodb.com/docs/atlas/atlas-search/atlas-search-overview/#search-nodes-architecture) with [`mongodbatlas_search_deployment`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_deployment).
+- You can now leverage [`type`](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Atlas-Search/operation/createAtlasSearchIndex) and [`fields`](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Atlas-Search/operation/createAtlasSearchIndex) parameters in [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) resource.
+
+
+1.13.0 also includes other general improvements, bug fixes, and several key documentation updates. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
+
```

Comment:
from the [changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md#v1130-2023-11-21) I can't see any breaking change, deprecation or removal to include those sections here. I double checked with @AgustinBettati and it seems there isn't any. Let me know if there is and I will include it